src/main/scala/com/cognite/sdk/scala/v1/fdm/common/properties/PropertyDefinition.scala

```diff
@@ -43,33 +44,64 @@ object PropertyDefinition {
   ) extends ViewPropertyDefinition
       with CorePropertyDefinition
 
-  final case class ConnectionDefinition(
+  sealed trait ConnectionDefinition extends ViewPropertyDefinition
+
+  final case class ReverseDirectRelationConnection(
+      name: Option[String],
+      description: Option[String],
+      connectionType: ReverseDirectRelationConnectionType,
+      source: ViewReference,
+      through: ThroughConnection
+  ) extends ConnectionDefinition
+
+  final case class ThroughConnection(
```

Comment:
nit: docs describe it as `The view or container of the node containing the direct relation property.`
So not sure if it is through "connection" (which also initially was looking as if it is a 3rd case for ConnectionDefinition), maybe some other name would fit better, ThroughReference, ThroughDefinition or something

Reply:
Fair! Renamed it to throughReference